portfolio/src/main/java/com/google/sps/util/NicknameUtil.java

```diff
@@ -8,11 +8,12 @@
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 
-
- public class NicknameUtil {
- /** Returns the nickname of the user with id, or empty String if the user has 
-    not set a nickname.
-    */
+  /** Static utilities for checking user nicknames. */
+public class NicknameUtil {
+ /** 
+  * Returns the nickname of the user with id, or empty String if the user has 
+  * not set a nickname.
+  */
```

Comment:
nit: `/**` should left align with the method indentation.
```suggestion
  /** 
   * Returns the nickname of the user with id, or empty String if the user has 
   * not set a nickname.
   */
```